content/authoring/tutorials/create-first-kata.mdx

```diff
@@ -32,175 +32,192 @@ To create a new kata, you need to earn the ["Create Kata" privilege](/gamificati
-- Puzzles
+- Fundamentals - Core language and API knowledge.
+- Algorithms - Logic required to complete the task.
+- Bug Fixes - Analyzing existing code, identifying the issue, and fixing it.
+- Refactoring - Rewriting and improving existing code.
+- Puzzles - Extrapolating a solution from a limited set of information.
 
 #### Estimated rank
 
-Here you specify how difficult you think your kata is. This value is used as an initial estimate for the kata difficulty and will be refined while collecting rank votes from other users who solved it. Your perception of the difficulty of your kata might differ from what the community might think and estimate, so your kata may end up approved with a higher or lower rank.
+This is where you specify how difficult you think your kata is. This value is used as an initial estimate for the kata's difficulty and will be refined by collecting rank votes from other users who solved it. Your perception of your kata's difficulty might differ from that of the community, so your kata may end up approved with a higher or lower rank than your initial choice.
 
 #### Tags
 
-Tags are used to classify your kata and make it easier to search for. Some tags are derived from the assigned discipline and added automatically, but you can add more.
+Tags are used to classify your kata and make it easier to find via search. Some tags are derived from the assigned discipline and added automatically, but you can add more if you like.
 
-#### Allow Contributors?
+#### Allow Contributors
 
-If you allow others to contribute to your kata, they will be able to make changes to it when it's a subject of the [beta process](/concepts/kata/beta-process/). However, even when you do not allow for contributors, sufficiently privileged users will be able to edit your kata if it has some pending issues or requires fixing.
+You can allow other users to contribute and make changes to your kata while it undergoes the [beta process](/concepts/kata/beta-process/). However, even when you do not allow contributors, certain users with sufficient privilege will still be able to edit your kata if it has pending issues or requires fixing.
 
 You can check the list of [privileges](/gamification/privileges/#privileges) to see who can modify your kata.
 
 ### Description
 
-The description field is used to provide instructions to the users who will be training on the kata. This field recognizes [Github flavored markdown](/references/markdown/) with a set of [Codewars extensions](/references/markdown/extensions/). You can use the `Preview` tab to see how it will look like when presented to users.
+The description field is where you provide the information and instructions users require to train on your kata. This field recognizes [Github flavored markdown](/references/markdown/) with a set of [Codewars extensions](/references/markdown/extensions/). Use the `Preview` tab to see how it will look like when presented to users.
```

Comment:
```suggestion
The description field is where you provide the information and instructions users require when training on your kata. This field recognizes [Github flavored markdown](/references/markdown/) with a set of [Codewars extensions](/references/markdown/extensions/). Use the `Preview` tab to see how it will look like when presented to users.
```